torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -1975,35 +1975,75 @@ int listMulIntRight(Stack& stack) {
   return 0;
 }
 
+// Stolen (with appropriate modifications) from cpython repo
```

Comment:
I think you might need to look into [Python's license](https://github.com/python/cpython/blob/master/LICENSE) before we can copy the code just like that. There are usually some restrictions like "you have to include this text along the sources".

Reply:
Looks like you can add `"Copyright (c) 2001, 2002, 2003, 2004, 2005, 2006, 2007, 2008, 2009, 2010,
2011, 2012, 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020 Python Software Foundation;
All Rights Reserved"` above the code and that is sufficient.